### PR TITLE
Using Github Actions in Feature Branch

### PR DESCRIPTION
**Pull request by normainobary2021**
Ability to use Linters in Github by adding Github Actions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
-# hello-world-4
\ No newline at end of file
+# hello-world-4
+
+> Description of the project.
+
+
```
Comment (algerina) [inline README.md]: - [ ] Kindly Use the Microverse [template ](https://github.com/microverseinc/readme-template)for your Readme file, and Microverse Logo before the title and the author's social media links are missing.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @normainobary2021 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- You have meaningful commits
- Your project looks good

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Adding a descriptive title and description to your pull request.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,67 @@
...
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```
Comment (Ugboaja-Uchechi) [inline README.md]: - [ ] [OPTIONAL] Kindly remove all headers in the README that isn't necessary so your README can look more professional.

File: README.md [hunk 3]
```diff
@@ -1 +1,67 @@
...
+## Built With
+
+- HTML, CSS
+- Linter
+- Github Actions
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
```
Comment (Ugboaja-Uchechi) [inline README.md]: - [ ] [OPTIONAL] Kindly remove the live demo since it isn't necessary for this project.

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @normainobary2021 , 

## STATUS APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- Great commit messages ✔️ 
- Repo looks professional ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (normainobary2021) [reply README.md]: Alright please. Will do.

**Pull request merged**